addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -254,22 +259,32 @@ registerModel({
          * @private
          */
         _prepareAttachmentsLoading() {
-            this._isPreparingAttachmentsLoading = true;
-            this._attachmentsLoaderTimeout = this.env.browser.setTimeout(() => {
-                this.update({ isShowingAttachmentsLoading: true });
-                this._isPreparingAttachmentsLoading = false;
-            }, this.messaging.loadingBaseDelayDuration);
+            this.update({
+                _attachmentsLoaderTimeout: this.env.browser.setTimeout(
+                    this._onAttachmentsLoadingTimeout,
+                    this.messaging.loadingBaseDelayDuration,
+                ),
+                _isPreparingAttachmentsLoading: true,
+            });
         },
         /**
          * @private
          */
         _stopAttachmentsLoading() {
             this.env.browser.clearTimeout(this._attachmentsLoaderTimeout);
-            this._attachmentsLoaderTimeout = null;
-            this._isPreparingAttachmentsLoading = false;
+            this.update({
+                _attachmentsLoaderTimeout: clear(),
+                _isPreparingAttachmentsLoading: clear(),
+            });
         },
     },
     fields: {
+        _attachmentsLoaderTimeout: attr({
+            default: null,
```

Comment:
there is a need for `null` instead of `undefined`?

Reply:
Just to keep same values as before turning them to fields. But we can use `undefined` instead of `null`.